code/ship/ship.h

```diff
@@ -1545,11 +1540,9 @@ extern int ship_query_general_type(ship *shipp);
 extern int ship_docking_valid(int docker, int dockee);
 extern int get_quadrant(vec3d *hit_pnt, object *shipobjp = NULL);	//	Return quadrant num of given hit point.
 
-extern void ship_obj_list_rebuild();	// only called by save/restore code
 extern int ship_query_state(char *name);
 
 // Goober5000
```

Comment:
Remove the `// Goober5000` comment too, because I didn't write the secondary function.